core/src/main/java/org/literacybridge/core/spec/Language.java

```diff
@@ -0,0 +1,46 @@
+package org.literacybridge.core.spec;
+
+import java.text.ParseException;
+import java.util.Map;
+
+public class Language {
+    final static String[] FILENAMES = new String[] { "pub_languages.csv", "languages.csv", "language_spec.csv" };
```

Comment:
As with ProjectsHelper.java, "pub_languages.csv" is new, so there can't be an old "languages.csv" or "language_spec.csv".